src/Products/PlonePAS/setuphandlers.py

```diff
@@ -196,11 +200,25 @@ def setupAuthPlugins(portal, pas, plone_pas,
         login_path = crumbler.auto_login_page
         cookie_name = crumbler.auth_cookie
 
-    found = uf.objectIds(['Extended Cookie Auth Helper'])
-    if not found:
-        plone_pas.manage_addExtendedCookieAuthHelper('credentials_cookie_auth',
-                                                     cookie_name=cookie_name)
-    logger.debug("Added Extended Cookie Auth Helper.")
+    is_plone_site = plone_ifaces.IPloneSiteRoot.providedBy(portal)
+    if is_plone_site:
+        cookie_meta_type = cookie_handler.ExtendedCookieAuthHelper.meta_type
+        add_cookie_plugin = plone_pas.manage_addExtendedCookieAuthHelper
+    else:
+        # Can't use the `ExtendedCookieAuthHelper` outside of a Plone portal.
+        cookie_meta_type = CookieAuthHelper.CookieAuthHelper.meta_type
+        add_cookie_plugin = pas.addCookieAuthHelper
+    cookie_auth_ids = uf.objectIds(cookie_meta_type)
+    if not cookie_auth_ids:
+        add_cookie_plugin(
+            "credentials_cookie_auth",
+            cookie_name=cookie_name,
+        )
+        logger.debug(
+            "Added %r: %r",
+            cookie_meta_type,
+            "/".join(uf.credentials_cookie_auth.getPhysicalPath()),
+        )
```

Comment:
f-strings preferred in Python 3 land. Anyway, minor one.
```suggestion
        logger.debug(
            f"Added {cookie_meta_type}: "
            f"{'/'.join(uf.credentials_cookie_auth.getPhysicalPath()}"
        )
```

Reply:
I've still been using `%` string formatting for logging calls, @jensens, because the arguments passed into the `logger.{level}(msg, args)` are actually recorded in the `args` attribute on the `LogRecord` which may be used by `Handler`s, `Filter`s, etc..  Very minor issue, but it also postpones interpolation until after decisions have been made about whether to emit a given record.  IOW, under normal operation the interpolation for a `DEBUG` level record such as this case will never even happen if we use `%` string formatting but would always happen if we use `f"..."` strings.

So I prefer keeping it as is.  If you still disagree and this will block merging, feel free to unresolve and LMK and I'll make and push the change.

Reply:
Just to capture some other context here in case anyone else runs across this, I recently ran across [the `Use %s formatting in logging functions` warning from PyLint](https://pycodequ.al/docs/pylint-messages/w1203-logging-fstring-interpolation.html) which both demonstrates agreement from at least some part of the Python code quality community and points out another reason not to use f-strings for logging calls, namely lazy evaluation.  IOW, the work of interpolating strings isn't done *unless* the given logging record will actually be logged.  For example, one can feel free to do a bunch of rich interpolation in `DEBUG` records without fear that they'll incur a performance penalty under normal operation.